### PR TITLE
Conditional ci

### PR DESCRIPTION
**Pull request by orangewolf**

### CODE REVIEW

File: .github/workflows/build-test-lint.yaml [hunk 1]
```diff
@@ -21,15 +21,11 @@ jobs:
...
-    needs: build
-    uses: scientist-softserv/actions/.github/workflows/test.yaml@v0.0.3
-    if: github.repository == 'scientist-softserv/harvard-curiosity'
-    with:
-      worker: false
+      # tag: deca90d9
   lint:
     needs: build
-    uses: scientist-softserv/actions/.github/workflows/lint.yaml@v0.0.3
+    uses: scientist-softserv/actions/.github/workflows/lint.yaml@main
```
Comment (bkiahstroud) [inline .github/workflows/build-test-lint.yaml:27]: Can we target a tag, or does this have to be `main`?

File: .github/workflows/build-test-lint.yaml [hunk 2]
```diff
@@ -21,15 +21,11 @@ jobs:
...
     if: github.repository == 'scientist-softserv/harvard-curiosity'
     with:
       worker: false
+      # tag: deca90d9
```
Comment (bkiahstroud) [inline .github/workflows/build-test-lint.yaml:31]: Were these "tag" comments added on purpose? If so, why?

File: .github/workflows/build-test-lint.yaml (hunk 1 above)
Comment (orangewolf) [reply .github/workflows/build-test-lint.yaml:27]: once we have it working, we can point it at a new tag

File: .github/workflows/build-test-lint.yaml (hunk 2 above)
Comment (orangewolf) [reply .github/workflows/build-test-lint.yaml:31]: they are a nice reminder that you can select a tag and skip the build. I just leave them as a placeholder for when I need it next

**Review by bkiahstroud — APPROVED** [review]

**Pull request merged**